CorinneTest/Person.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+namespace CorinneTest
+{
+    public class Person 
+    {
+        private string name;
+        public string Name
+        {
+            get => this.name; 
+            set { this.name = value; }
+        }
+
+        private int? age;
+        public int? Age
+        {
+            get => this.age;
+            set {
+                if (value < 1) value = 1;
+                else if (value > 100) value = 100;
+                this.age = value;
+            }
+        }
+
+        private Gender gender;
+        public Gender Gender
+        {
+            get => this.gender; 
+            set { this.gender = value; }
+
+        }
+
+        private DateTime? birthDate;
+        public DateTime? BirthDate
+        {
+            get => this.birthDate; 
+        }
+
+        public Person(string myName, int? myAge, Gender myGender, DateTime? myBirthDate)
```

Comment:
Generally constructors should be the first thing in your class after private field declaration.

Reply:
Also, should you do any parameter validation here?  What happens if `myName` is `null` or empty?  Is that ok?

Reply:
As nick said, coding cleanliness usually puts things in this order:

- Variables
- Constructor
- Instance Methods
- Then whatever else

Reply:
so Person constructor after private variable declarations and then the getter/setter methods?

Reply:
@nnieslan